### ISSUE TITLE
训练的过程map总是0

### ISSUE THREAD

**Issue by WhXl** [TomMao23/multiyolov5#7]
## ❔Question
train: Scanning '/home/wx/dataset/multiyolov5/fod/detdata/labels/train.cache' images and labels... 11 found, 0 missing, 2 empty, 0 corrupted: 100%|██████████| 11/11 [01:19<?, ?it/s]
100%|██████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 2/2 [00:00<00:00,  2.40it/s]
    39/999     5.31G    0.1613   0.03633         0    0.1977   0.02364         9       832: 100%|██████████████████████████████████████████████████████| 2/2 [00:00<00:00,  2.40it/s]
               Class      Images      Labels           P           R      mAP@.5  mAP@.5:.95: 100%|████████████████████████████████████████████████████| 2/2 [00:00<00:00,  6.99it/s]
                 all          10           0           0           0           0           0

## Additional context
查看日志像是标签未加载！
我的任务是检测1类，语义分割加上背景为3类

**Comment by WhXl** [comment]
Optimizer groups: 77 .bias, 79 conv.weight, 73 other
train: Scanning '/home/wx/dataset/multiyolov5/fod/detdata/labels/train.cache' images and labels... 11 found, 0 missing, 2 empty, 0 corrupted: 100%|██████████| 11/11 [00:00<?, ?it/s]
train: Scanning '/home/wx/dataset/multiyolov5/fod/detdata/labels/train.cache' images and labels... 11 found, 0 missing, 2 empty, 0 corrupted: 100%|██████████| 11/11 [00:00<?, ?it/s]
Plotting labels... 
val: Scanning '/home/wx/dataset/multiyolov5/fod/detdata/labels/val.cache' images and labels... 10 found, 0 missing, 4 empty, 0 corrupted: 100%|██████████████| 10/10 [00:00<?, ?it/s]
train: Scanning '/home/wx/dataset/multiyolov5/fod/detdata/labels/train.cache' images and labels... 11 found, 0 missing, 2 empty, 0 corrupted: 100%|██████████| 11/11 [00:00<?, ?it/s]

autoanchor: Analyzing anchors... anchors/target = 5.84, Best Possible Recall (BPR) = 1.0000
Found 10 images in the folder /home/wx/dataset/multiyolov5/customdata/segimages/val
BaseDataset: base_size 832, crop_size (832, 832)
Random scale low: 0.75, high: 1.5, sample_std: 35
Found 10 images in the folder /home/wx/dataset/multiyolov5/customdata/segimages/train
Image sizes 832 train, 832 test
Using 4 dataloader workers
Logging results to runs/train/exp11
Starting training for 1000 epochs...
accumulate: 16

     Epoch   gpu_mem       box       obj       cls     total       seg    labels  img_size
100%|██████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 2/2 [00:04<00:00,  2.28s/it]
     0/999     5.32G    0.1656    0.0468         0    0.2124    0.3807         7       832: 100%|██████████████████████████████████████████████████████| 2/2 [00:04<00:00,  2.28s/it]
pixAcc: 0.979, mIoU: 0.052: 100%|████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 10/10 [00:00<00:00, 11.83it/s]
               Class      Images      Labels           P           R      mAP@.5  mAP@.5:.95: 100%|████████████████████████████████████████████████████| 2/2 [00:01<00:00,  1.44it/s]
                 all          10           0           0           0           0           0
accumulate: 1

     Epoch   gpu_mem       box       obj       cls     total       seg    labels  img_size
100%|██████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 2/2 [00:00<00:00,  2.15it/s]
     1/999     5.31G    0.1651   0.04519         0    0.2103    0.2437         9       832: 100%|██████████████████████████████████████████████████████| 2/2 [00:00<00:00,  2.15it/s]
               Class      Images      Labels           P           R      mAP@.5  mAP@.5:.95: 100%|████████████████████████████████████████████████████| 2/2 [00:00<00:00,  4.45it/s]
                 all          10           0           0           0           0           0
accumulate: 1

     Epoch   gpu_mem       box       obj       cls     total       seg    labels  img_size
100%|██████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 2/2 [00:00<00:00,  2.09it/s]
     2/999     5.31G      0.16   0.04658         0    0.2066    0.1424         6       832: 100%|██████████████████████████████████████████████████████| 2/2 [00:00<00:00,  2.09it/s]
               Class      Images      Labels           P           R      mAP@.5  mAP@.5:.95: 100%|████████████████████████████████████████████████████| 2/2 [00:00<00:00,  4.45it/s]
                 all          10           0           0           0           0           0
accumulate: 1

     Epoch   gpu_mem       box       obj       cls     total       seg    labels  img_size
100%|██████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 2/2 [00:00<00:00,  2.10it/s]
     3/999     5.31G    0.1598   0.04654         0    0.2063   0.08955         4       832: 100%|██████████████████████████████████████████████████████| 2/2 [00:00<00:00,  2.10it/s]
               Class      Images      Labels           P           R      mAP@.5  mAP@.5:.95: 100%|████████████████████████████████████████████████████| 2/2 [00:00<00:00,  4.49it/s]
                 all          10           0           0           0           0           0
accumulate: 1

     Epoch   gpu_mem       box       obj       cls     total       seg    labels  img_size
100%|██████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 2/2 [00:00<00:00,  2.17it/s]
     4/999     5.31G    0.1642   0.04325         0    0.2074   0.06059         5       832: 100%|██████████████████████████████████████████████████████| 2/2 [00:00<00:00,  2.17it/s]
               Class      Images      Labels           P           R      mAP@.5  mAP@.5:.95: 100%|████████████████████████████████████████████████████| 2/2 [00:00<00:00,  4.48it/s]
                 all          10           0           0           0           0           0

**Comment by WhXl** [comment]
我解决了这个问题，是因为样本训练样本太少了
